src/Shared/FrameworkLocationHelper.cs

```diff
@@ -559,7 +559,7 @@ internal static string GetPathToDotNetFramework(Version version, DotNetFramework
         /// Check the registry key and value to see if the .net Framework is installed on the machine.
         /// </summary>
         /// <param name="registryEntryToCheckInstall">Registry path to look for the value</param>
-        /// <param name="registryValueToCheckInstall">Key to retrieve the value from</param>
+        /// <param name="registryValueToCheckInstall">Key to retrieve the value from</param>
```

Comment:
I'm assuming this (and the other handful of visibly unchanged lines) are EOL fixes as mentioned in #22 ?